database/insertData/05_review_assignments.js

```diff
@@ -8,9 +8,14 @@ exports.queries = [
       input: {
         reviewAssignment: {
           applicationId: 4000
-          assignerId: 8
-          reviewerId: 6
           stageId: 5
+          userToReviewerId: {
+            connectByUsername: {
+              username: "testReviewer1"
+            }
```

Comment:
Love how we can connect by username since it's a unique field. Wish we had more of that TBH.

Reply:
Yes, it's so nice that way.